### DISCUSSION TITLE
User Filament Printer profile storage path

### DISCUSSION THREAD

**Discussion by yexele** [OrcaSlicer/OrcaSlicer#3167] [General]
Does anyone know where to find the user filament profiles you save go?  I can find the stock profiles but not the custom ones?

**Comment by LJKraus** [comment]
Help>Show Configuration folder.
Open the "user" folder, then the folder with a number for a name.

**Comment by yexele** [comment]
Thanks I found it my computer had it double hidden On Dec 23, 2023 3:18 PM, LJKraus ***@***.***> wrote:
Help>Show Configuration folder.
Open the "user" folder, then the folder with a number for a name.

**Comment by Rhaletskyj** [comment]
thx
